sci-physics/qutip/qutip-4.1.2.ebuild

```diff
@@ -0,0 +1,47 @@
+# Copyright 1999-2017 Gentoo Foundation
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=6
+
+PYTHON_COMPAT=( python2_7 python3_{5,6} )
+inherit distutils-r1 toolchain-funcs
+
+DESCRIPTION="Simulating dynamics of open quantum systems in Python."
+HOMEPAGE="http://qutip.org"
+SRC_URI="https://github.com/qutip/qutip/archive/v$PV.tar.gz -> $P.tar.gz"
+
+LICENSE="BSD"
+SLOT="0"
+KEYWORDS="~amd64 ~x86"
+IUSE="openmp test"
+
+RDEPEND="dev-python/cython[${PYTHON_USEDEP}]
+	dev-python/numpy[${PYTHON_USEDEP}]
+	dev-python/matplotlib[${PYTHON_USEDEP}]
+	sci-libs/scipy[${PYTHON_USEDEP}]
+"
+DEPEND="${RDEPEND}
+	test? ( dev-python/nose[${PYTHON_USEDEP}] )
+"
+
+pkg_pretend() {
+	use openmp && tc-check-openmp
+}
```

Comment:
handling OpenMP is a bit more tricky:

```
pkg_pretend() {
	[[ ${MERGE_TYPE} != binary ]] && use openmp && tc-check-openmp
}

pkg_setup() {
	[[ ${MERGE_TYPE} != binary ]] && use openmp && tc-check-openmp
}
```
Yes, you *need* to check for it in `pkg_setup`, and you *may* check for it in `pkg_pretend`